test/test_functional_tensor.py

```diff
@@ -1024,7 +1024,9 @@ def test_gaussian_blur(device, image_size, dt, ksize, sigma, fn):
     #     "23_23_1.7": ...
     # }
     p = os.path.join(os.path.dirname(os.path.abspath(__file__)), "assets", "gaussian_blur_opencv_results.pt")
-    true_cv2_results = torch.load(p)
+
+    # Potentially unsafe `weights_only=False` is needed here.
+    true_cv2_results = torch.load(p, weights_only=False)
```

Comment:
As explained above, there is nothing unsafe here. We are loading from a static asset that is part of our repository.

```suggestion
    true_cv2_results = torch.load(p, weights_only=False)
```